client/src/main/scala/io/delta/sharing/spark/DeltaSharingSource.scala

```diff
@@ -1124,7 +1125,10 @@ case class DeltaSharingSource(
       }
       Some(v)
     } else if (options.startingTimestamp.isDefined) {
-      Some(deltaLog.client.getTableVersion(deltaLog.table, options.startingTimestamp))
+      val version = deltaLog.client.getTableVersion(deltaLog.table, options.startingTimestamp)
+      logInfo(s"Got table version $version for timestamp ${options.startingTimestamp} " +
+        s"from Delta Sharing Server.")
+      Some(version)
     } else {
       None
     }
```

Comment:
Can this change be unit tested?

Reply:
there's no real code change, just added a logInfo.

Reply:
Aren't we changing the user agent header being sent by this client?
Can we test that the user agent as various fields? (perhaps a test was missing earlier).

Reply:
I see, taking a look.

Reply:
added